addons/account_asset/views/account_asset_view.xml

```diff
@@ -161,17 +160,33 @@
                                     </group>
                                 </form>
                             </field>
-                            <button type="object" name="compute_depreciation_board" string="Compute" icon="terp-stock_format-scientific" colspan="2" attrs="{'invisible':[('state','=','close')]}"/>
                         </page>
                         <page string="History">
-                            <field name="account_move_line_ids" readonly="1"/>
                             <field name="history_ids" readonly="1"/>
                         </page>
-                        <page string="Notes">
-                            <field name="note" placeholder="Add an internal note here..."/>
-                        </page>
                     </notebook>
+<!--                     <separator string="Depreciation Information" colspan="4"/>
+                    <group>
+                        <group>
+                            <field name="method" widget="radio" attrs="{'invisible': [('type','=','sale')]}"/>
+                            <field name="method_progress_factor" attrs="{'invisible':[('method','=','linear')], 'required':[('method','=','degressive')]}"/>
+                            <label for="method_time"/>
+                            <div>
+                                <field name="method_time"/>
+                            </div>
+                            <field name="prorata" attrs="{'invisible': [('method_time','=','end')]}"/>
+                        </group>
+                        <group>
+                            <field name="method_number" attrs="{'invisible':[('method_time','=','end')], 'required':[('method_time','=','number')]}"/>
+                            <field name="method_period"/>
+                            <field name="method_end" attrs="{'required': [('method_time','=','end')], 'invisible':[('method_time','=','number')]}"/>
+                        </group>
+                    </group> -->
```

Comment:
BST wanted to hide this Depreciation Information section, but we never know who can change his mind, so commented code ! Should I remove it ?